ui/src/app/views/app/component.ts

```diff
@@ -56,6 +58,11 @@ export class AppComponent implements OnInit {
       }
     });
     this.user = this.signInService.user;
+    this.profileService.getMe().subscribe(profile => {
```

Comment:
See above comment. This is fine for a placeholder, but I'm not sure that having administrators log into the main workbench UI is the right UX for this feature.